src/pages/events.js

```diff
@@ -47,6 +47,9 @@ export const EventsContent = styled.div`
   display: grid;
   grid-template-rows: auto auto;
   grid-template-columns: 1fr;
+  min-height: 90vh;
+  align-items: start;
+  align-content: start;
```

Comment:
have used `align-content` and `align-items` here coz event contents are taking extra space it gets from height and centering/spacing themselves(idk better way to explain this verbally lol )